code/modules/telesci/rcs.dm

```diff
@@ -117,6 +117,24 @@
 		spark(src, 5)
 		to_chat(user, "<span class = 'caution'>You emag the RCS. Click on it to toggle between modes.</span>")
 
+	else if(W.is_screwdriver(user))
+		if(cell)
+			cell.updateicon()
+			cell.forceMove(get_turf(src.loc))
```

Comment:
keep forceMove, tht's good, but also add put_in_hands after it so it tries to go into the user's hand, you can get rid of src. here too

this has the slightly quirky outcome that if you are inside a locker and you remove the battery, it will drop outside the locker instead of into the locker, since you are opting for the floor